src/prog/BSMPT.cpp

```diff
@@ -32,7 +32,7 @@
 using namespace std;
 using namespace BSMPT;
 
-struct CLIOptions
+struct CLIOptions_BSMPT
```

Comment:
Why the rename?

Reply:
Breathe complained about duplicate definitions.